src/main/java/cl/transbank/patpass/Options.java

```diff
@@ -0,0 +1,55 @@
+package cl.transbank.patpass;
+
+import cl.transbank.common.Option;
+import com.google.gson.annotations.SerializedName;
+import lombok.*;
+import lombok.experimental.Accessors;
+
+@ToString @NoArgsConstructor @AllArgsConstructor
+public class Options extends Option implements Cloneable  {
+    @Getter  final String headerCommerceCodeName = "commercecode";
+    @Getter  final String headerApiKeyName = "Authorization";
+    @Setter @Getter @Accessors(chain = true) private String commerceCode;
```

Comment:
`commerce_code` and `api_key` could be inherited from `Options` so you could delete them from here.

Reply:
will do